decidim-core/app/models/decidim/notification.rb

```diff
@@ -7,6 +7,9 @@ class Notification < ApplicationRecord
     belongs_to :resource, foreign_key: "decidim_resource_id", foreign_type: "decidim_resource_type", polymorphic: true
     belongs_to :user, foreign_key: "decidim_user_id", class_name: "Decidim::User"
 
+    scope :daily, ->(time: Time.now.utc) { where(created_at: time.beginning_of_day..time.end_of_day) }
```

Comment:
This should be from yesterday, something like 
```suggestion
    scope :daily, ->(time: Time.now.utc - 1.day) { where(created_at: time.beginning_of_day..time.end_of_day) }
```